src/main/java/com/cloudblue/connect/api/clients/utils/URLBuilder.java

```diff
@@ -0,0 +1,48 @@
+package com.cloudblue.connect.api.clients.utils;
```

Comment:
This file is no longer needed as we have implemented url builder in Q.

Reply:
Done